### PR TITLE
Add MiniKF guide for Kubeflow

### PR DESCRIPTION
**Pull request by cspavlou**
Add instructions for installing MiniKF, as presented at Kubeflow Day LA at SCaLE 17x.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/534)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
Preview: https://deploy-preview-534--competent-brattain-de2d6d.netlify.com/docs/started/getting-started-minikf/

File: content/docs/started/getting-started-minikf.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+for the official announcement and why we started MiniKF.
+
+Join the discussion on the
+[#minikf](https://kubeflow.slack.com/messages/CGRKM3N0G/) Slack channel,
+ask questions, request features, and get support for MiniKF.
+
+### System requirements
+For a smooth experience we recommend that your system meets the
+following requirements:
+  - 12GB RAM
```
Comment (sarahmaddox) [inline content/docs/started/getting-started-minikf.md]: The bullet formatting is not working - see preview:
https://deploy-preview-534--competent-brattain-de2d6d.netlify.com/docs/started/getting-started-minikf/

Adding an empty line before the first list item should solve this.

File: content/docs/started/getting-started-minikf.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  - 2 CPUs
+  - 50GB disk space
+
+### Operating systems
+MiniKF runs on all major operating systems:
```
Comment (sarahmaddox) [inline content/docs/started/getting-started-minikf.md]: Same here about bullet formatting.

File: content/docs/started/getting-started-minikf.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+### Kubeflow dashboard
+You can access the Kubeflow dashboard by opening a browser and going to:
+
+```
+http://10.10.10.10
+```
+
+### How to log in
+Use these credentials to log in to all services, e.g., JupyterHub, Rok:
```
Comment (sarahmaddox) [inline content/docs/started/getting-started-minikf.md]: s/e.g./for example

Tech writer hint: Always spell out “i.e.” as “that is”, and “e.g.” as “for example,”
Reason: Very many people don't know the difference between "e.g." and "i.e.". The resulting confusion can be quite bad. Also, abbreviations are hard to parse, particularly for people whose first language is other than English.

File: content/docs/started/getting-started-minikf.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+installs in minutes.
+
+MiniKF is a fast and easy way to get started with Kubeflow. With just a
+few clicks, you are up for experimentation, and for running complete
+Kubeflow Pipelines.
+
+To train at scale, move to a Kubeflow cloud deployment with one click,
+without having to rewrite anything.
+
+See
```
[the remark this answers is not in the document]
Comment (cspavlou) [reply content/docs/started/getting-started-minikf.md]: Thanks for the hint :)

File: content/docs/started/getting-started-minikf.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+[here](https://medium.com/kubeflow/minikf-the-fastest-and-easiest-way-to-deploy-kubeflow-on-your-laptop-a91fb846d0ba)
+for the official announcement and why we started MiniKF.
```
[the remark this answers is not in the document]
Comment (cspavlou) [reply content/docs/started/getting-started-minikf.md]: I changed the sentence, thanks

File: content/docs/started/getting-started-minikf.md (hunk 1 above)
Comment (cspavlou) [reply content/docs/started/getting-started-minikf.md]: Added an empty line, thank you!

File: content/docs/started/getting-started-minikf.md (hunk 3 above)
Comment (cspavlou) [reply content/docs/started/getting-started-minikf.md]: Thank you for the hint, I appreciate it

**Comment by sarahmaddox** [comment]
Nice.

/ok-to-test
/lgtm
/approve
